apps/condo/domains/common/components/containers/YandexMetrika.tsx

```diff
@@ -1,40 +1,41 @@
-import React from 'react'
+import { useRouter } from 'next/router'
+import { useEffect } from 'react'
 import getConfig from 'next/config'
 
+import ym, { YMInitializer } from 'react-yandex-metrika'
+
 const YandexMetrika = () => {
     const { publicRuntimeConfig } = getConfig()
     const { yandexMetrikaID } = publicRuntimeConfig
 
+    const router = useRouter()
+
+    const routeChangeComplete = () => {
+        ym('hit', window.location.pathname)
```

Comment:
Please check the case on first render. In case that we use next with SSR enabled `window` object might be undefined here